src/edu/csus/ecs/pc2/core/util/XMLMemento.java

```diff
@@ -1,11 +1,11 @@
-// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+// Copyright (C) 1989-20259 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
```

Comment:
Typo in updated copyright date (20259 -> 2025)

Reply:
Fixed in next push.